src/stores/OverlayStore.ts

```diff
@@ -973,7 +910,11 @@ export class OverlayColorbarSettings {
     }
 
     @computed get numberWidth(): number {
-        const textWidth = Math.max(...this.texts.map(x => x.length)) * this.textRatio[clamp(Math.floor(this.numberFont / 4), 0, this.textRatio.length)];
+        const activeFrame = AppStore.Instance.activeFrame;
+        if (!activeFrame) {
+            return 0;
+        }
+        const textWidth = Math.max(...activeFrame.colorbarStore.texts.map(x => x.length)) * this.textRatio[clamp(Math.floor(this.numberFont / 4), 0, this.textRatio.length)];
```

Comment:
The changes about colorbar looks good to me 👍 
Just found one possible enhancement: we can use the max of `Math.max(...activeFrame.colorbarStore.texts.map(x => x.length))` of all 1~4 frames, instead of just checking the active frame. The `numberWidth` will not be changing when we click different panels in this way